src/pyj/read_book/paged_mode.pyj

```diff
@@ -615,10 +616,10 @@ def scroll_to_extend_annotation(backward):
 
 def handle_shortcut(sc_name, evt):
     if sc_name is 'up':
-        scroll_by_page(True, True)
+        scroll_by_page(backward=True, by_screen=True, flip_if_rtl_page_progression=False)
```

Comment:
I added named parameters for all of these because it was getting really hard to read the logic with a series of three booleans.